db/migrate/20151210225310_create_test_client_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTestClientRequests < ActiveRecord::Migration
```

Comment:
Definitely meant to leave it in.

I created a generic client implementation in the `Test` namespace. It needs to be an actual ActiveRecord class in order to test the ES integration. Since we had been talking about creating/documenting what it takes to create a new Client model, I thought this would kill the proverbial 2 fowl.